pkg/component/observability/monitoring/prometheus/longterm/scrapeconfigs.go

```diff
@@ -56,7 +56,11 @@ func CentralScrapeConfigs() []*monitoringv1alpha1.ScrapeConfig {
 						`{__name__=~"garden_shoot_info:timestamp:this_month"}`,
 						`{__name__=~"metering:(cpu_requests|memory_requests|network|persistent_volume_claims|disk_usage_seconds|memory_usage_seconds).*:this_month"}`,
 						`{__name__="garden_shoot_node_info"}`,
-						`{__name__="garden_shoot_condition", condition="APIServerAvailable"}`,
+						`{__name__="garden_shoot_condition", condition=("APIServerAvailable"|"SystemComponentsHealthy")}`,
+						`{__name__="garden_seed_condition", condition=("SeedSystemComponentsHealthy"|"GardenletReady")}`,
```

Comment:
Unfortunately it seems that we missed to test this in the local setup :(

The syntax

```
`{__name__="garden_shoot_condition", condition=("APIServerAvailable"|"SystemComponentsHealthy")}`,
```

is kind of obviously incorrect and the author probably meant

```
`{__name__="garden_shoot_condition", condition="(APIServerAvailable|SystemComponentsHealthy)"}`,
```

and with that the federation to the long term Prometheus was inadvertently but completely broken.

For the future we should probably qualify the `/lgtm` response, e.g.:

- it looks good to me, but I actually did not test it, just skimmed over the source code
- it looks good to me and I'm confident that it does not cause regressions because ...

Unfortunately our integration and end-to-end tests can not catch such regressions: the garden conditions do not cover the state of each Prometheus scrape job, which is OK. We could brainstorm how this could be improved in a meaningful way.

The unit tests kind of repeat the syntax error that managed to slip through the review and `/lgtm` process, although the full diff is only 10 lines.

cc @rickardsjp @vicwicker